host/xtest/rand_stream.c

```diff
@@ -71,10 +65,8 @@ static void get_random(struct rand_stream *rs, uint8_t *buf, size_t blen)
 		b += t;
 
 		if (rs->w_offs == sizeof(rs->word_buf)) {
-			int32_t r;
-
-			random_r(&rs->random_data, &r);
-			memcpy(rs->word_buf, &r, sizeof(r));
+			rs->seed = (rs->seed * 1103515245 + 12345);
```

Comment:
What does the ` * 1103515245 + 12345` add here? I.e, will the seed become a better seed by doing this?